testutils/generators_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/block-explorer/blob/master/LICENSE.md.
+
+package testutils
+
+import (
+	"io"
+	"testing"
+
+	"github.com/insolar/insolar/ledger/heavy/exporter"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGenerateRecords(t *testing.T) {
+	batchSize := 5
+	f := GenerateRecords(batchSize)
+
+	n := uint32(1)
+	for i := 0; i < batchSize; i++ {
+		record, err := f()
+		require.NoError(t, err)
+		require.Equal(t, n, record.RecordNumber)
+		n++
+	}
+	res, err := f()
```

Comment:
record, err := f() and res, err := f() is different, because each function call generates NEW records.